packages/atomic/src/components/recommendations/atomic-recs-list/e2e/atomic-recs-list.e2e.ts

```diff
@@ -101,6 +101,24 @@ test.describe('when there are no enough recommendations for multiple pages', ()
   });
 });
 
+test.describe('when recommendations open in a new tab', async () => {
+  test.beforeEach(async ({recsList}) => {
+    await recsList.load({story: 'recs-opening-in-new-tab'});
+    await recsList.hydrated.waitFor();
+  });
+
+  test.only('should open a single tab when clicking a recommendation', async ({
```

Comment:
Surprised that nothing flagged this `.only` up. :x:

Reply:
Good catch! Completely forgot to remove it.